.gitignore

```diff
@@ -160,3 +160,7 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+# vscode
+.history/
+.vscode/
+*.code-workspace
```

Comment:
@FuzzkingCool this may be better to add into your global `.gitignore` file so that you wouldn't need to put these in git repos you're working in.

For now, @iLLiCiTiT thoughts of including this?

Reply:
No problem including them, there is `.idea` too, but it should be inbetween `Mac Stuff` and `JetBrain` so IDEs are together, also make sure there is an empty line between them, it looks like vscode is part of mac stuff now.

Reply:
☝️